src/watchdog/tricks/__init__.py

```diff
@@ -177,60 +179,106 @@ class AutoRestartTrick(Trick):
 
     def __init__(self, command, patterns=None, ignore_patterns=None,
                  ignore_directories=False, stop_signal=signal.SIGINT,
-                 kill_after=10):
+                 kill_after=10, debounce_interval_seconds=0):
         super().__init__(
             patterns=patterns, ignore_patterns=ignore_patterns,
             ignore_directories=ignore_directories)
+
+        if kill_after < 0:
```

Comment:
Those checks could be moved above the previous line.

Reply:
Done.